### DISCUSSION TITLE
How do I implement a geo location chooser for mobile devices?

### DISCUSSION THREAD

**Discussion by Zerachiel01** [unoplatform/uno#18045] [Q&A] [answered]
Hello everyone!

I am searching for a way to implement a control that lets the user pick any geo location existing in for example google maps.
I am thinking of something like this on google maps:

![image](https://github.com/user-attachments/assets/8b7930ce-b991-440f-b4f3-4394a8da54dd)

Is there an existing control for uno platform that I couldn't find? Did anyone else try to achieve something like this?

Thanks in advance!

**Comment by MartinZikmund** [comment, marked as answer]
@Zerachiel01 you will need to use some external API as the source for the data (e.g. Google Maps API). Then you can surface such autosuggest experience via the `AutoSuggestBox` control that is built into Uno Platform.

**Reply by Zerachiel01** [reply]
I expected something like this, thanks for the quick answer!
